src/os_db.ml

```diff
@@ -14,8 +16,8 @@ let one f ~success ~fail q = f q >>= function r :: _ -> success r | _ -> fail
 
 let pwd_crypt_ref =
   ref
-    ( (fun password -> Bcrypt.string_of_hash (Bcrypt.hash password))
-    , fun _ password1 password2 ->
+    ( (fun password -> Bcrypt.string_of_hash (Bcrypt.hash password)),
+      fun _ password1 password2 ->
```

Comment:
Did you ocamlformat with our .ocamlformat?

Reply:
Maybe. In that case we should probably remove it from the repository, isn't it?

Reply:
`os_db.ml` is uptodate ! This is ready to be merged.